### PR TITLE
Syslog security filter: add 'ident' value to logged message

### PR DESCRIPTION
**Pull request by mgladi**

### CODE REVIEW

File: source/code/plugins/filter_syslog_security.rb [hunk 1]
```diff
@@ -36,7 +36,7 @@ def filter(tag, time, record)
       # The time may be slightly in the future from the ingestion time.
       record['Timestamp'] = OMS::Common.format_time(Time.now.to_f)
 
-      record['Message'] = record['message']
+      record['Message'] = record['ident'] + ': ' + record['message']
       record.delete 'message'
       record.delete 'time'
```
Comment (robbiezhang) [inline source/code/plugins/filter_syslog_security.rb]: delete 'ident' too.

**Comment by robbiezhang** [comment]
Signoff

File: source/code/plugins/filter_syslog_security.rb (hunk 1 above)
Comment (varunkumta) [inline source/code/plugins/filter_syslog_security.rb]: Can you explain why you need to do this? The 'ident' field is ingested into search as the 'ProcessName' field

Comment (mgladi) [inline source/code/plugins/filter_syslog_security.rb]: We are parsing a subset of the messages sent by the agent in the workflow. The parsing extracts relevant fields from the message (source/dest ip, protocol, action etc.). For messages that are not parsed we'd like to provide the full message to the users. 
There is no ProcessName field in the type to which the messages are mapped (CommonSecurityLog)

The 'ident' field is also required

**Pull request merged**